### PR TITLE
I'm getting an error while trying to use Mineflayer, this should help

### PR DESCRIPTION
**Pull request by javaveryhot**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -60,12 +60,12 @@ module.exports = {
...
-  options.username = options.username ?? 'Player'
-  options.version = options.version ?? false
-  options.plugins = options.plugins ?? {}
-  options.hideErrors = options.hideErrors ?? true
-  options.logErrors = options.logErrors ?? true
-  options.loadInternalPlugins = options.loadInternalPlugins ?? true
+  options.username = options.username || 'Player'
+  options.version = options.version || false
+  options.plugins = options.plugins || {}
+  options.hideErrors = options.hideErrors || true
```
Comment (Moondarker) [inline index.js:66]: Please read [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
hideErrors, logErrors and loadInternalPlugins will be forever true

**Review by Moondarker — CHANGES_REQUESTED** [review]

**Comment by Moondarker** [comment]
Also, if you are getting an error, sharing it in such PR (or in issues first) would be a great idea

**Comment by BlueBurgersTDD** [comment]
Having a error here is because your node is old, update it

**Comment by Karang** [comment]
Update node, mineflayer support node 14+

**Pull request closed without merging**